pages/schedule/index.tsx

```diff
@@ -312,13 +314,13 @@ export default function Calendar(props: { scheduleCard: ScheduleEvent[] }) {
                   </p>
                   <p>{eventData.time}</p>
                 </div>
-                {/* <div className="">
```

Comment:
Once again mostly a nit, 

Avoid commenting out code, git exists so we can get old code back. Having commended out code makes the diff more difficult to parse and the code harder to read

Reply:
I'm not 100% sure where this came from, but I can't see it anymore on my end.